Создание страницы прямого перевода

mrowa223 opened the pull request:
feat: add direct page, 
transactions with the same username 
![image](https://github.com/user-attachments/assets/11fb698c-9b51-47b4-b355-b1d03405e543)

mrowa223 committed:
feat: add transaction direct to user page (status 500)

mrowa223 committed:
feat: add direct to user page

efValeron reviewed (changes requested):
Добавь страничку подтверждения транзакции, я тебе в тг написал

efValeron committed:
Delete package-lock.json

Сейчас стоит другой пакетный менеджер - bun. package-lock не принимается. Больше я думаю не нужно будет пакеты устанавливать, так что package-lock не комитить
